spi/src/main/java/com/amazon/opendistroforelasticsearch/jobscheduler/spi/ScheduledJobParameter.java

```diff
@@ -72,4 +72,14 @@ default Long getLockDurationSeconds() {
      * @return job execution jitter
      */
     default Double getJitter() {return null;}
+
+    /**
+     * The job will be delayed for a fixed amount of time for every execution.
+     * It's a simple way for the job to catch out or order (delayed or late) events most of the time.
```

Comment:
Can you explain more about this in comments? How to use fixed delay to catch delayed/late event?
My understanding: custom job runner needs to get job expected execution time rather than current time to run query. For example, ingestion job always delay 5min, the job expected to run at 1:00AM ideally, considering the ingestion delay, we set "delaySeconds" as 300s to catch latest records which should arrive before 1:00AM (but it will arrive at 1:05AM as ingestion event is late). So the job will execute at 1:05AM actually, and the query will still query data before 1:00AM(expected execution time). We can get expected execution time via `JobExecutionContext#getExpectedExecutionTime`

Reply:
I think we can remove the comment about late events.. that's specific to the plugin use case and this could be used for other reasons.

Reply:
Agree, if we add the comments for the late event "It's a simple way for the job to catch out or order (delayed or late) events most of the time.", we should add more details. Either remove this comment for late event, or put more details about this use case.

Reply:
Sorry that I didn't notice the reply in time in the morning. Thanks for the ideas, and I will remove the comments for the specific use case, and try to state the general meaning.

Reply:
Removed the line that describes a specific use case.